apisix/cli/schema.lua

```diff
@@ -209,9 +209,6 @@ local config_schema = {
                         ssl_trusted_certificate = {
                             type = "string",
```

Comment:
Do we also need to set system to the json schema default?

Reply:
I think if we add it to apisix/cli/config.lua. Its fine to not add this here.

Reply:
I still recommend adding it so that we can explore the feasibility of generating documents for it in the future.
Although we don't need it to fill us with default values, as a specification it should fit the status quo.